spec/test.js

```diff
@@ -0,0 +1,36 @@
+import 'babel-polyfill';
```

Comment:
isn't that deprecated? I found that regenerator-runtime was sufficient for this purpose.